Util/HttplugFactory.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace Http\HttplugBundle\Util;
```

Comment:
I do not like the `Util` namespace. It feels like the old days when you had a folder called `functions`. Maybe do something more descriptive like `Discovery`?